comps/comps-foreman-el8.xml

```diff
@@ -405,8 +405,8 @@
       <packagereq type="default">rubygem-google-apis-pubsub_v1-doc</packagereq>
       <packagereq type="default">rubygem-google-apis-sqladmin_v1beta4-doc</packagereq>
       <packagereq type="default">rubygem-google-apis-storage_v1-doc</packagereq>
-      <packagereq type="default">rubygem-googleauth-doc</packagereq>
```

Comment:
We know this from the past, Mac does sorting a bit different then Linux, can you keep the order and only add the new package(s)?

Reply:
Reverted sorting and only added my package.